src/devtools/client/debugger/src/components/Editor/EmptyLines.tsx

```diff
@@ -27,21 +28,27 @@ const connector = connect(mapStateToProps);
 type PropsFromRedux = ConnectedProps<typeof connector>;
 
 interface ELProps {
-  editor: any;
+  editor: SourceEditor;
 }
 
 type FinalELProps = PropsFromRedux & ELProps;
 
 class EmptyLines extends Component<FinalELProps> {
+  _rafId: number | null = null;
```

Comment:
It took me a minute to realize `raf` was `requestAnimationFrame`. It's a mouthful, so I am not sure I want to change the variable name, and I realized it pretty quick, but thought I'd mention it in case anyone else has good suggestions for another name